functions/billing/main.py

```diff
@@ -36,7 +36,7 @@
 
 # [START functions_billing_limit]
 # [START functions_billing_stop]
-PROJECT_ID = os.getenv('GCP_PROJECT')
+PROJECT_ID = os.getenv('PROJECT_ID')
```

Comment:
N.B: this value will need to be set explicitly on deployment. (`GCP_PROJECT` used to be auto-populated in older runtimes.)

Reply:
Yeah, I'm gonna roll back that change and update the documentation to be more clear around it.